### ISSUE TITLE
Error and segfault when loading superpixels and probabilities in multicut

### ISSUE THREAD

**Issue by snajder-r** [ilastik/ilastik#1844]
Hi!

I am trying to load superpixels in the multicut workflow. The weird thing is:

* Loading raw + superpixels works
* Loading raw + probabilities works
* Loading raw + probabilities + superpixels doesn't work!

If I do the latter, I get an error message saying the dataset could not be loaded and in the log I get the following error:

```
ERROR 2018-09-19 14:31:23,595 log_exception 72407 140633780938560 Traceback (most recent call last):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/ilastik/ilastik/applets/dataSelection/dataSelectionGui.py", line 602, in _configureOpWithInfos
    self.topLevelOperator.DatasetGroup[laneIndex][roleIndex].setValue( info )
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 118, in call_in_setup_context
    return func(self, *args, **kwargs)
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1229, in setValue
    self._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1430, in _changed
    c._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1430, in _changed
    c._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1430, in _changed
    c._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1436, in _changed
    self._configureOperator(self)
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1450, in _configureOperator
    self.operator._setupOutputs()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/operator.py", line 459, in _setupOutputs
    self.setupOutputs()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/ilastik/ilastik/applets/dataSelection/opDataSelection.py", line 459, in setupOutputs
    self.Image.connect(providerSlot)
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 118, in call_in_setup_context
    return func(self, *args, **kwargs)
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 554, in connect
    self._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1430, in _changed
    c._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1430, in _changed
    c._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1430, in _changed
    c._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1436, in _changed
    self._configureOperator(self)
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1450, in _configureOperator
    self.operator._setupOutputs()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/operator.py", line 459, in _setupOutputs
    self.setupOutputs()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/operators/valueProviders.py", line 396, in setupOutputs
    self.reset()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/operators/valueProviders.py", line 421, in reset
    self.Output.connect(self.PrecomputedInput)
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 118, in call_in_setup_context
    return func(self, *args, **kwargs)
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 554, in connect
    self._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1430, in _changed
    c._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1430, in _changed
    c._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1430, in _changed
    c._changed()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1436, in _changed
    self._configureOperator(self)
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/slot.py", line 1450, in _configureOperator
    self.operator._setupOutputs()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/operator.py", line 459, in _setupOutputs
    self.setupOutputs()
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/ilastik/ilastik/applets/edgeTraining/opEdgeTraining.py", line 199, in setupOutputs
    assert self.Superpixels.meta.dtype == np.uint32
AssertionError
```

While it is true that my superpixels are not uint32 (I tried uint8 and float), it is odd that this would be  a requirement when ground truth can be uint8.  Or at the very least the error message could inform me about this requirement. It would also be nice if this would be checked right after a load the file, instead of loading and displaying it correctly and only throwing an error once the lane is ready.

What's even worse is that after clicking "OK" on the error message, it segfaults on CentOS 7 (not on Windows, however). The error before the segfault is below:

```
ERROR 2018-09-19 14:31:23,595 log_exception 72407 140633780938560 Wasn't able to load your dataset into the workflow.  See error log for details.
*** Error in `/g/kreshuk/snajder/ilastik-1.3.0-Linux/bin/python': malloc(): memory corruption (fast): 0x0000000004dbeb1f ***
======= Backtrace: =========
/lib64/libc.so.6(+0x7f5e4)[0x7fe7d932c5e4]
/lib64/libc.so.6(+0x82d00)[0x7fe7d932fd00]
/lib64/libc.so.6(__libc_malloc+0x4c)[0x7fe7d933284c]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libxcb.so.1(+0xe2fb)[0x7fe7bf9662fb]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libxcb.so.1(+0xfa78)[0x7fe7bf967a78]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libxcb.so.1(+0xca00)[0x7fe7bf964a00]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libxcb.so.1(+0xe8c7)[0x7fe7bf9668c7]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libxcb.so.1(xcb_wait_for_reply+0x6d)[0x7fe7bf9669e0]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libxcb.so.1(xcb_translate_coordinates_reply+0x28)[0x7fe7bf970d5e]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/plugins/platforms/../../lib/libQt5XcbQpa.so.5(_ZNK10QXcbScreen10topLevelAtERK6QPoint+0x52)[0x7fe7be7e3162]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Gui.so.5(_ZN15QGuiApplication10topLevelAtERK6QPoint+0x1a7)[0x7fe7c8993357]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN12QApplication10topLevelAtERK6QPoint+0x9)[0x7fe7c8ff3f19]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN12QApplication8widgetAtERK6QPoint+0x16)[0x7fe7c8ff4526]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN19QApplicationPrivate14sendMouseEventEP7QWidgetP11QMouseEventS1_S1_PS1_R8QPointerIS0_Eb+0x5b7)[0x7fe7c8ffbb67]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(+0x1ba164)[0x7fe7c9054164]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(+0x1bc1b3)[0x7fe7c90561b3]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent+0x9c)[0x7fe7c8ff7b5c]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN12QApplication6notifyEP7QObjectP6QEvent+0x3fa)[0x7fe7c8ffc1da]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/QtWidgets.so(+0x3543de)[0x7fe7c98d73de]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Core.so.5(_ZN16QCoreApplication15notifyInternal2EP7QObjectP6QEvent+0x75)[0x7fe7c85c0325]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Gui.so.5(_ZN22QGuiApplicationPrivate17processMouseEventEPN29QWindowSystemInterfacePrivate10MouseEventE+0x40b)[0x7fe7c89996db]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Gui.so.5(_ZN22QGuiApplicationPrivate24processWindowSystemEventEPN29QWindowSystemInterfacePrivate17WindowSystemEventE+0x115)[0x7fe7c899ae75]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Gui.so.5(_ZN22QWindowSystemInterface22sendWindowSystemEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE+0x7b)[0x7fe7c897c64b]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/plugins/platforms/../../lib/libQt5XcbQpa.so.5(+0xa04d0)[0x7fe7be8344d0]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libglib-2.0.so.0(g_main_context_dispatch+0x254)[0x7fe7c359a614]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libglib-2.0.so.0(+0x4a868)[0x7fe7c359a868]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libglib-2.0.so.0(g_main_context_iteration+0x2c)[0x7fe7c359a90c]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Core.so.5(_ZN20QEventDispatcherGlib13processEventsE6QFlagsIN10QEventLoop17ProcessEventsFlagEE+0x5c)[0x7fe7c861188c]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Core.so.5(_ZN10QEventLoop4execE6QFlagsINS_17ProcessEventsFlagEE+0xfb)[0x7fe7c85be43b]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN5QMenu4execERK6QPointP7QAction+0x64)[0x7fe7c91762a4]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(+0x2ebd14)[0x7fe7c9185d14]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Core.so.5(_ZN11QMetaObject8activateEP7QObjectiiPPv+0x2c1)[0x7fe7c85ea7d1]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(+0x25842d)[0x7fe7c90f242d]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN15QAbstractButton15mousePressEventEP11QMouseEvent+0xa6)[0x7fe7c90f2a66]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/QtWidgets.so(+0x3298fb)[0x7fe7c98ac8fb]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN7QWidget5eventEP6QEvent+0x983)[0x7fe7c903a643]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/QtWidgets.so(+0x32a3bb)[0x7fe7c98ad3bb]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN19QApplicationPrivate13notify_helperEP7QObjectP6QEvent+0x9c)[0x7fe7c8ff7b5c]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN12QApplication6notifyEP7QObjectP6QEvent+0xc6d)[0x7fe7c8ffca4d]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/QtWidgets.so(+0x3543de)[0x7fe7c98d73de]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Core.so.5(_ZN16QCoreApplication15notifyInternal2EP7QObjectP6QEvent+0x75)[0x7fe7c85c0325]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(_ZN19QApplicationPrivate14sendMouseEventEP7QWidgetP11QMouseEventS1_S1_PS1_R8QPointerIS0_Eb+0x1a0)[0x7fe7c8ffb750]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5(+0x1b9bc1)[0x7fe7c9053bc1]
/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/site-packages/PyQt5/../../../libQt5Widgets.so.5Fatal Python error: Aborted

Thread 0x00007fe7237fe700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 299 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 551 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 1180 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Thread 0x00007fe7459de700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 299 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/operators/cacheMemoryManager.py", line 241 in _wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/operators/cacheMemoryManager.py", line 160 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Thread 0x00007fe77b7fe700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 295 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 213 in _get_next_job
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 176 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Thread 0x00007fe77bfff700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 295 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 213 in _get_next_job
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 159 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Thread 0x00007fe790de9700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 295 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 213 in _get_next_job
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 176 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Thread 0x00007fe7915ea700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 295 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 213 in _get_next_job
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 159 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Thread 0x00007fe791deb700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 295 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 213 in _get_next_job
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 159 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Thread 0x00007fe7925ec700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 295 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 213 in _get_next_job
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 176 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Thread 0x00007fe792ded700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 295 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 213 in _get_next_job
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 176 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Thread 0x00007fe7935ee700 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 295 in wait
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 213 in _get_next_job
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/lazyflow/lazyflow/request/threadPool.py", line 176 in run
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 916 in _bootstrap_inner
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/lib/python3.6/threading.py", line 884 in _bootstrap

Current thread 0x00007fe7da8e6740 (most recent call first):
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/ilastik/ilastik/shell/gui/startShellGui.py", line 70 in startShellGui
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/ilastik/ilastik_main.py", line 152 in main
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/ilastik/ilastik.py", line 108 in main
  File "/g/kreshuk/snajder/ilastik-1.3.0-Linux/ilastik-meta/ilastik/ilastik.py", line 122 in <module>
/g/kreshuk/snajder/ilastik-1.3.0-Linux/run_ilastik.sh: line 68: 72407 Aborted                 (core dumped) "${PREFIX}/bin/python" "${PREFIX}/ilastik-meta/ilastik/ilastik.py" "$@"
```

Ilastik version 1.3.0

**Comment by DerThorsten** [comment]
Having a uint32 requirement for super-pixels is very reasonable.
Super-pixels are by definition an over-segmentation. To have an actual over-segmentation one usually needs more than 256 superpixels (which would be the max for uint8).

**Comment by k-dominik** [comment]
thanks @DerThorsten for the insights! :) Highly appreciated.

**Comment by glancybp** [comment]
I had the same issue yesterday.  Changing the superpixels to uint32 fixed the problem.  However, I agree that it would be nice to have this requirement stated somewhere, perhaps on the Ilastik Multicut documentation webpage.

**Comment by k-dominik** [comment]
related (for the segfault): #2968
